### ISSUE TITLE
about the training loss curve

### ISSUE THREAD

**Issue by Ideal-Bushgorman** [360CVGroup/PlanGen#3]
Hi, thanks for your great work!
Would it be possible to share the training loss curve or logs to better understand how the model converges? It would be very helpful for reproducing the results.

**Comment by C0notSilly** [comment]
Also wondering about the loss curve. In my case, when working on some AIGC tasks, the loss stays relatively high at the early stage of the training. However, with the loss continuing to decrease, the trained model collapses.

@Devil-Ideal Did you encounter a similar issue in fine-tuning Janus-Pro 1.5B?

**Comment by hrz2000** [comment]
Sorry for the late reply. PlanGen's training is relatively stable and simple, its text part next-token prediction loss decreases faster and more obviously, for the image part, the fluctuation is relatively large, just like the diffusion loss, this is the reference of the training curve:

![Image](https://github.com/user-attachments/assets/9b078a62-7c5e-4e7d-8ea3-df9828b81d63)
